music21/mei/test_base.py

```diff
@@ -3897,7 +3897,7 @@ def testMeasureIntegration1(self):
         # ensure the right number and @n of parts
         self.assertEqual(4, len(actual.keys()))
         for eachN in expectedNs:
-            self.assertTrue(eachN in actual.keys())
+            self.assertTrue(eachN in actual)
```

Comment:
Please, let's get out of the habit of putting fixes unrelated to the issue at hand in the same PR.  My amount of review time is severely constrained in the first month or so of term and I want to make sure I can get through as many in half a day on Saturdays as I can.

Reply:
Wholeheartedly agree; I try to stay hygienic about such things. This happened because this was the first branch to fail CI on a new pylint version, and I think this part of the diff would have disappeared if I had, after those changes were merged, pulled again from master? Sorry! I'll keep the branches up to date!